ui/src/components/Pagination.tsx

```diff
@@ -112,10 +112,29 @@ export default function Pagination(props: PaginationProps) {
             key={i}
             page={page}
             currentPage={currentPage}
-            onPageChange={onPageChange}
+            onPageChange={(page) => onPageChange(page, pageSize)}
           />
         ))}
       </div>
+      <div className="hidden md:-mt-px md:flex">
+        <select
+          className="text-gray-500 mt-4 inline-flex items-center border-transparent text-sm font-medium focus:border-transparent focus:ring-0"
```

Comment:
![CleanShot 2023-12-06 at 17 28 40](https://github.com/flipt-io/flipt/assets/209477/8a8fd195-e0f2-4799-83e7-958e32f760e8)

styling looks good I think we can always iterate in the future. I think we need to set a bg color for the select box so that it renders correctly in darkmode. something like bg-gray-100 will do the trick i think

Reply:
Or bg-gray-50. We use a similar style for the combo box for namespaces dropdown I think